extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -572,23 +583,27 @@ export class AmpSidebar extends AMP.BaseElement {
    * @private
    */
   setupGestures_(element) {
+    if (this.disableSwipeClose_) {
+      return;
+    }
     // stop propagation of swipe event inside amp-viewer
     const gestures = Gestures.get(
       dev().assertElement(element),
-      /* shouldNotPreventDefault */ false,
+      /* shouldNotPreventDefault */ true,
       /* shouldStopPropagation */ true
     );
     gestures.onGesture(SwipeXRecognizer, (e) => {
-      const {data} = e;
-      this.handleSwipe_(data);
+      const {data, event} = e;
```

Comment:
Isn't e the event? Can you add a comment explaining this?

Reply:
Thanks, yep comment added in latest.